Doc/library/locale.rst

```diff
@@ -345,22 +351,26 @@ The :mod:`locale` module defines the following exception and functions:
    ``'LANG'``.  The GNU gettext search path contains ``'LC_ALL'``,
    ``'LC_CTYPE'``, ``'LANG'`` and ``'LANGUAGE'``, in that order.
 
-   Except for the code ``'C'``, the language code corresponds to :rfc:`1766`.
-   *language code* and *encoding* may be ``None`` if their values cannot be
+   The format of the language code is platform depended, but on Posix
+   platforms it usually looks like ``language[_territory]``.
+   The language code and encoding may be ``None`` if their values cannot be
```

Comment:
Please readd the mention of the RF: "looks like ``language[_territory]`` (see :rfc:`1766`)."

Reply:
Well, I think that it is worth to add a separate section for locale names, with all specifications and links.